activities-sticky-queues/activities.go

```diff
@@ -0,0 +1,60 @@
+package activities_sticky_queues
+
+import (
+	"context"
+	"crypto/sha256"
+	"io"
+	"os"
+	"time"
+
+	"go.temporal.io/sdk/activity"
+)
+
+// DownloadFile creates a file on the host with some test data.
+func DownloadFile(ctx context.Context, url, path string) (err error) {
+	logger := activity.GetLogger(ctx)
+	logger.Info("Downloading file", "url", url, "path", path)
+	// Create the file
+	out, err := os.Create(path)
+	if err != nil {
+		return
+	}
+	defer func() {
+		err = out.Close()
+	}()
+	time.Sleep(3 * time.Second)
+	_, err = out.WriteString("downloaded body")
```

Comment:
Consider `os.WriteFile` here unless it's really that important to create the file 3 seconds before you write it